example/pubspec.yaml

```diff
@@ -0,0 +1,14 @@
+name: example
+description: Demonstrates how to use the platform api.
+version: 1.0.0
+publish_to: none
+
+environment:
+  sdk: ^3.0.0
```

Comment:
This should probably use 3.2 like the package itself.

Reply:
Done